.github/workflows/ci.yml

```diff
@@ -80,6 +80,6 @@ jobs:
     - uses: actions/checkout@v4
     - uses: actions-rs/toolchain@v1
       with:
-        toolchain: "1.67"
+        toolchain: "1.72"
```

Comment:
`x509-certificate` -> `zeroize` released a new version(1.8.0) thirteen hours ago, now we require at least rustc `1.72`.